Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/YeeAlgorithm.H

```diff
@@ -0,0 +1,122 @@
+/* Copyright 2020 Remi Lehe
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef WARPX_FINITE_DIFFERENCE_ALGORITHM_YEE_H_
+#define WARPX_FINITE_DIFFERENCE_ALGORITHM_YEE_H_
+
+#include <AMReX_REAL.H>
+#include <AMReX_Array4.H>
+#include <AMReX_Gpu.H>
+
+struct YeeAlgorithm {
```

Comment:
```suggestion
/**
 * This struct contains only static functions to initialize the stencil coefficients
 * and to compute finite-difference derivatives to the Cartesian Yee FTDT algorithm.
 */
struct YeeAlgorithm {
```
Furthermore, for consistency, shouldn't this `struct` be named `YeeAlgorithm`? Or are you afraid that, to make it consistent, we would need a **lot** of changes?
This also applied to other FD algorithms.